web/src/styles/base/typography.scss

```diff
@@ -1,3 +1,14 @@
+@font-face {
```

Comment:
dust - we have some font limitations, what exactly are we introducing here? Are we using a different font weigh, if so the design system needs to be updated and the typography section of storybook. And this new font style needs to be globally availabile via a CSS name, note sure if it is

Reply:
@seidior implemented this portion of it, can you shed some light on what was your intention here?

Reply:
You're correct @FarazA22, this introduces the semibold weight, which is missing due to NJWDS's implementation of USWDS, which doesn't include semibold by default.